Change from react-testing-library to testing-library/react

LeviButcher opened the pull request:
As requested in #59 

Let me know of anything else needed for this merge.

LeviButcher commented:
@itaditya I've updated my branch to resolve conflicts with master. I forced push to prevent from creating many commits.

I realized the format command isn't running on the tests directory. Would it be okay to change it to
"prettier --write src/* src/**/*"?

itaditya commented:
That's a good catch @LeviButcher. Ideally a PR should focus only on one thing. Since prettier formatting is different from your work.

**But for now, you can keep the prettier change**.

itaditya commented on package.json:
```diff
@@ -10,18 +10,18 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "format": "prettier --write src/*"
+    "format": "prettier --write src/* src/**/*"
```
```suggestion
    "format": "prettier --write src/**/*"
```

just doing this must be enough

LeviButcher replied:
I'm on Ubuntu and having just src/**/* only found the tests directory. Wrapping it in single quotes seems to do the trick for getting everything.

LeviButcher commented:
@itaditya I've updated my branch with latest changes.
The format command was breaking trying to format the mp3 file so I changed it to match only .js.
But that's not a very good solution since we probably want css files formatted as well.

I can add in a css file match but want to make sure your fine with that and if there are any other patterns we should match on.

itaditya commented:
@LeviButcher prettier has the option to ignore files as well. See this https://prettier.io/docs/en/ignore.html#ignoring-files

So we can say ignore mp3 file via .prettierignore file

LeviButcher commented:
@itaditya  Good to go. :+1:

itaditya commented on .prettierignore:
```diff
@@ -0,0 +1 @@
+*.mp3
```
Add a newline at the end to make the red symbol go away.

Also configure your editor to add a newline to files automatically

itaditya commented:
Can you run these commands-

```
git pull upstream master # get the latest code of my master in your fork's branch
npm install # install the dependencies so it auto-generates correct package-lock.json
```

This will automatically fix the merge conflicts.

itaditya commented:
Congrats @LeviButcher 

![](https://media.giphy.com/media/ehhuGD0nByYxO/giphy.gif)
